tools/run_make_firmwares3.py

```diff
@@ -875,9 +882,15 @@ def __init__(self, target, target_D, extra_D_list, build_dir, elf_name):
     },{ 
         'target' : 'rx-R9MM-f103rb',                    'target_D' : 'RX_R9MM_868_F103RB',
         'extra_D_list' : [], 'appendix' : '' 
+    },{ 
+        'target' : 'rx-R9MM-f103rb',                    'target_D' : 'RX_R9MM_868_F103RB',
+        'extra_D_list' : ['MLRS_FEATURE_ELRS_BOOTLOADER'], 'appendix' : '-for-elrs-bl' 
```

Comment:
suggestion: I think it would be sufficient if the appendix would be just '-elrs-bl'. Makes it a bit less verbose und clunky.